certbot-dns-rfc2136/certbot_dns_rfc2136/_internal/dns_rfc2136.py

```diff
@@ -40,6 +40,7 @@ class Authenticator(dns_common.DNSAuthenticator):
     }
 
     PORT = 53
+    sign_query = "False"
```

Comment:
Please follow the existing pattern for defining credential options, have a look at `_setup_credentials` and `_validate_credentials`.